scenes/game/local_game.gd

```diff
@@ -5481,7 +5904,7 @@ func do_card_from_hand_to_gauge(performing_player : Player, card_ids : Array) ->
 	return true
 
 func do_boost_name_card_choice_effect(performing_player : Player, card_id : int) -> bool:
-	var card_name = card_db.get_card_name(card_id)
+	var card_name = "the card " + card_db.get_card_name(card_id)
 	if card_id == -1:
 		card_name = "a nonexistent card"
 	printlog("SubAction: BOOST_NAME_CARD by %s card %s" % [get_player_name(performing_player.my_id), card_name])
```

Comment:
by PLAYER card the card CARDNAME?
Gonna make the printlog look weird
though this is just for debug

Reply:
just changed it to "named %s" to help a bit